game/src/environment/towers/core/Tower.tscn

```diff
@@ -8,105 +8,102 @@
 [sub_resource type="RectangleShape2D" id=1]
 extents = Vector2( 64, 64 )
 
-[sub_resource type="CircleShape2D" id=2]
```

Comment:
Is this delete intended?

Reply:
see https://github.com/bharathcs/quantum-defence/pull/61#pullrequestreview-699530283 and https://github.com/bharathcs/quantum-defence/pull/62